packages/medusa/src/loaders/feature-flags/pricing-integration.ts

```diff
@@ -0,0 +1,10 @@
+import { FlagSettings } from "../../types/feature-flags"
+
+const PricingIntegrationFeatureFlag: FlagSettings = {
+  key: "pricing_integration",
+  default_val: false,
+  env_key: "MEDUSA_FF_PRICE_MODULE",
```

Comment:
suggestion(non-blocking): The key and env_key are usually very similar. We could also use one that would be consistent with the Product Module isolation flag?
```suggestion
  key: "isolate_pricing_domain",
  default_val: false,
  env_key: "MEDUSA_FF_ISOLATE_PRICING_DOMAIN",
```